ui/lib/utils/distroStringsRes.ts

```diff
@@ -0,0 +1,40 @@
+type DistroStringsRes = {
+  is_distro: boolean
+  tidb: string
+  tikv: string
+  pd: string
+  tiflash: string
+}
+
+const defalutDistroStringsRes: DistroStringsRes = {
+  is_distro: false,
+  tidb: 'TiDB',
+  tikv: 'TiKV',
+  pd: 'PD',
+  tiflash: 'TiFlash',
+}
```

Comment:
Is it possible to not having this ts definition (types and default values) at all while keep type-check working? In this PR, people needs to modify a lot of places in order to introduce a new distribution string field, while previously what people only need to do is modifying the go source code of `type DistributionResource struct`, writing new fields and default values, and everything will work like charm. They don't need to change here and there.

Reply:
I have an idea, let me try.